src/http/api/views/explore.view.integration.test.ts

```diff
@@ -0,0 +1,258 @@
+import { afterAll, beforeAll, beforeEach, describe, expect, it } from 'vitest';
+
+import type { Knex } from 'knex';
+
+import type { Account } from '@/account/account.entity';
+import { ExploreView } from '@/http/api/views/explore.view';
+import { createTestDb } from '@/test/db';
+import { createFixtureManager, type FixtureManager } from '@/test/fixtures';
+
+describe('ExploreView', () => {
+    let db: Knex;
+    let fixtureManager: FixtureManager;
+    let exploreView: ExploreView;
+
+    beforeAll(async () => {
+        db = await createTestDb();
+        fixtureManager = await createFixtureManager(db);
+    });
+
+    beforeEach(async () => {
+        await fixtureManager.reset();
+        exploreView = new ExploreView(db);
+    });
+
+    afterAll(async () => {
+        await db.destroy();
+    });
+
+    async function createTopic(name: string, slug: string) {
+        const [id] = await db('topics').insert({ name, slug });
+        return { id, name, slug };
+    }
+
+    async function addAccountToTopic(accountId: number, topicId: number) {
+        await db('account_topics').insert({
+            account_id: accountId,
+            topic_id: topicId,
+        });
+    }
+
+    describe('getAccountsInTopic', () => {
+        it('should return empty array for non-existent topic', async () => {
+            const [viewer] = await fixtureManager.createInternalAccount();
+
+            const { accounts, next } = await exploreView.getAccountsInTopic(
+                'non-existent-topic',
+                viewer.id,
+            );
+
+            expect(accounts).toHaveLength(0);
+            expect(next).toBeNull();
+        });
+
+        it('shoudl return expected fields for accounts', async () => {
```

Comment:
typo: `shoudl`

Reply:
Fixed in 6a551a809589ec5153f0b11ea2bf28faa843edf4